.drone.yml

```diff
@@ -0,0 +1,186 @@
+kind: pipeline
+type: vm
+name: testing
+platform:
+  os: linux
+  arch: amd64
+pool:
+  use: ubuntu
+
+steps:
+- name: lint
+  image: golang:1.18
+  pull: always
+  commands:
+  - go install github.com/golangci/golangci-lint/cmd/golangci-lint@latest
+  - golangci-lint version
+  - golangci-lint run
+  volumes:
+  - name: gopath
+    path: "/go"
+- name: test
+  image: golang:1.18
+  commands:
+  - go test -cover ./...
+  volumes:
+  - name: gopath
+    path: "/go"
+volumes:
+- name: gopath
+  temp: {}
+trigger:
+  ref:
+  - refs/heads/master
+  - refs/tags/**
+  - refs/pull/**
+
+---
+kind: pipeline
+type: vm
+name: linux-amd64
+platform:
+  os: linux
+  arch: amd64
+pool:
+  use: ubuntu
+
+steps:
+- name: environment
+  image: golang:1.18
+  pull: always
+  environment:
+    CGO_ENABLED: '0'
+  commands:
+  - go version
+  - go env
+- name: build
+  image: golang:1.18
+  environment:
+    CGO_ENABLED: '0'
+  commands:
+  - go build -v -ldflags "-X main.version=" -a -tags netgo -o release/linux/amd64/drone-sftp .
```

Comment:
This is currently not correctly outputting the version but I'm not sure if that's still necessary in later Golang versions. If not then there should be a `build` and `build-tag` step that are conditional.